[BUG]: Builds fail on CI/CD

### What happened?

Tailwindcss was removed in favor of NextUI, [commit](https://github.com/developersIndia/website/commit/2e724c904908c689d58405a79f05536c079ffea7).

However, the imports are still present in [styles/globals.css](https://github.com/developersIndia/website/blob/master/src/styles/globals.css). 
```
@tailwind base;
@tailwind components;
@tailwind utilities;
```

Along with that, `postcss` and its [config](https://github.com/developersIndia/website/blob/master/postcss.config.js) and, [tailwindcss config](https://github.com/developersIndia/website/blob/master/tailwind.config.js) is still present, which again causes the issue.
```
module.exports = {
  plugins: {
    tailwindcss: {},
    autoprefixer: {},
  },
};
```

Which causes the CI/CD builds to fail

### What browsers are you seeing the problem on?

_No response_

### Relevant log output

```shell
Error: Cannot find module 'tailwindcss'

Import trace for requested module:
./src/styles/globals.css
./src/pages/_app.tsx
```


### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

--- a/README.md
+++ b/README.md
@@ -22,6 +22,7 @@ Add more entries of the tech stack as the project grows & developers over time -
 - [Next.js](https://nextjs.org/)
 - [TypeScript](https://www.typescriptlang.org)
 - [Vercel](https://vercel.com)
+- [NextUI](https://nextui.org
 
 ## 🏃 Getting Started
 

--- a/README.md
+++ b/README.md
@@ -22,7 +22,6 @@ Add more entries of the tech stack as the project grows & developers over time -
 - [Next.js](https://nextjs.org/)
 - [TypeScript](https://www.typescriptlang.org)
 - [Vercel](https://vercel.com)
-- [TailwindCSS](https://tailwindcss.com)
 
 ## 🏃 Getting Started
 

--- a/postcss.config.js
+++ b/postcss.config.js
@@ -1,6 +1,0 @@
-module.exports = {
-  plugins: {
-    tailwindcss: {},
-    autoprefixer: {},
-  },
-};

--- a/src/styles/globals.css
+++ b/src/styles/globals.css
@@ -1,3 +1,0 @@
-@tailwind base;
-@tailwind components;
-@tailwind utilities;

--- a/tailwind.config.js
+++ b/tailwind.config.js
@@ -1,8 +1,0 @@
-module.exports = {
-  content: [
-    "./src/**/*.{html,js,ts,jsx,tsx}",
-    "./src/**/*.{html,js,ts,jsx,tsx}",
-  ],
-  theme: { extend: {} },
-  plugins: [],
-};

